src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Selectors/GenericSecurityTokenAuthenticator.cs

```diff
@@ -21,7 +21,7 @@ protected override bool CanValidateTokenCore(SecurityToken token)
         protected override ReadOnlyCollection<IAuthorizationPolicy> ValidateTokenCore(SecurityToken token)
         {
             var genericToken = (GenericSecurityToken)token;
-            return Security.SecurityUtils.CreatePrincipalNameAuthorizationPolicies(genericToken.Name);
+            return CoreWCF.Security.SecurityUtils.CreatePrincipalNameAuthorizationPolicies(genericToken.Name);
         }
```

Comment:
Can you merge the two SecurityUtils classes together so that we only have the one. Otherwise we end up with a lot of duplicate implementation,

Reply:
Should I move the IdentityModel.SecurityUtils to Security.SecurityUtils ?

Reply:
Yes. There's some overlap in duplicated functionality between them, but some methods do the same thing but have subtle differences that I didn't want to spend the time unpicking. So I think the approach needed is combine them into Security.SecurityUtils if there isn't two versions of the method/class, and if there is, only combine if they are 100% equivalent. I don't think we have anything here which falls into that category at the moment, but it's possible there is.